miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -750,16 +754,6 @@ protected LibraryDto packageDtoFromParent(Sample sample) {
       }
       dto.setParentSampleId(sample.getId());
       dto.setParentSampleAlias(sample.getAlias());
-      if (namingScheme.hasLibraryAliasGenerator()) {
-        try {
-        Library tempLibrary = new LibraryImpl();
-        tempLibrary.setSample(sample);
-        final String libraryAlias = namingScheme.generateLibraryAlias(tempLibrary);
-        dto.setAlias(libraryAlias);
-        } catch (MisoNamingException e) {
-          log.error("Failed to generate library name", e);
-        }
-      }
```

Comment:
@frankie13 had set it up this way because that's how it was done before, but said it wasn't a requirement. Generating after save has the advantage of preventing duplicates if multiple people are creating libraries at the same time. When we have one to many propagation, generating before save gets even more problematic, and wouldn't have worked as it was.